### DISCUSSION TITLE
Welcome to PRIMO_watchdog Discussions!

### DISCUSSION THREAD

**Discussion by marcelinohermida** [marcelinohermida/PRIMO_WatchDog#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MP-YBX** [comment]
Can you record a complete usage operation video

**Comment by MP-YBX** [comment]
<img width="1047" alt="联想截图_20241219214500" src="https://github.com/user-attachments/assets/65214299-cf13-4b41-9f67-12da659bc2f9" />
I don't know to choose a PSF file,there are too many files
